demo/src/components/SrlComponent.js

```diff
@@ -239,92 +239,18 @@ class SrlInput extends React.Component {
   <SrlOutput /> Component
 *******************************************************************************/
 
-// Render the SRL tag for a single word as a table cell
-class SrlTagCell extends React.Component {
-
-  render() {
-    const { tag, colorClass } = this.props;
-
-    // Don't show "O" tags, and slice off all the "B-" and "I-" prefixes.
-    const tagText = tag === "O" ? "" : tag.slice(2);
-
-    return (
-      <td className={colorClass + ' srl-tag srl-tag-' + tag.toLowerCase()}>
-        {tagText}
-      </td>
-    )
-  }
-}
-
-// Render a SRL-tagged word as a table cell
-class SrlWordCell extends React.Component {
-  render() {
-    const { word, colorClass } = this.props;
-
-    return (<td className={colorClass + ' srl-word'}>{word}</td>)
-  }
-}
-
-class SrlFrame extends React.Component {
-  render() {
-    const { verb, words } = this.props;
-    const tags = verb["tags"];
-
-
-    // Skip frames that have only one tag; these are typically helper verbs.
-    // In an ideal world we'd filter these out on the backend, but the POS
-    // tagger we're using right now doesn't seem up to the task.
-    const numTags = tags.filter(tag => tag !== "O").length
-    if (numTags <= 1) {
-      return (<div />)
-    }
-
-    // Create an array indicating what color to highlight each tag cell.
-    // For "O" tags this should be -1, indicating no color.
-    // Otherwise it should toggle between 0 and 1 every time a "B-" tag occurs.
-    var colorClasses = [];
-    var currentColor = 1;
-
-    tags.forEach(function (tag, i) {
-      if (tag === "O") {
-        // "O" tag, so append "" for "no color"
-        colorClasses.push("");
-      } else if (tag[0] === "B") {
-        // "B-" tag, so toggle the current color and then append
-        currentColor = (currentColor + 1) % 2;
-        colorClasses.push("color" + currentColor);
-      } else /* (tag[0] == "I") */ {
-        // "I-" tag, so append the current color
-        colorClasses.push("color" + currentColor);
-      }
-    })
-
-    return (
-      <div>
-        <label>{verb.verb}</label>
-        <table className="srl-table">
-          <tbody>
-            <tr>
-              {tags.map((tag, i) => <SrlTagCell tag={tag} key={i} colorClass={colorClasses[i]} />)}
-            </tr>
-            <tr>
-              {words.map((word, i) => <SrlWordCell word={word} key={i} colorClass={colorClasses[i]} />)}
-            </tr>
-          </tbody>
-        </table>
-      </div>
-    )
-  }
-}
-
 class SrlOutput extends React.Component {
   render() {
-    const { words, verbs } = this.props;
+    const { verbs } = this.props;
 
     return (
       <div className="model__content model__content--srl-output">
-        <div className="form__field">
-          {verbs.map((verb, i) => (<SrlFrame verb={verb} words={words} key={i} />))}
+        <div>
+          {verbs.map((verb, i) => {
+              return (
+                  <p key={i}><b>{verb.verb}:</b> {verb.description}</p>
```

Comment:
Using the index as a key has one risk -- if you change the item at that index, without updating the entire array (i.e. updating the reference to a new array), React won't re-render (you won't see the effect of the change).  That said, this likely isn't an issue with the demo.

If it is and/or could be, I would use the verb + index:

```
key={`${i}-${verb.verb}`}
```

That said as noted, likely nothing to worry about, as I'm pretty sure there's no mutations going on here.